client/src/components/Hero.module.css

```diff
@@ -9,8 +9,8 @@
     display: flex;
     flex-wrap: wrap;
     align-items: center;
```

Comment:
Bootstrap has a utility class for it 'align-items-center'

https://getbootstrap.com/docs/4.0/utilities/flex/#align-items

Reply:
Done.